### PR TITLE
Add skip package feature

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

File: nixpkgs_review/cli/__init__.py [hunk 1]
```diff
@@ -156,6 +156,20 @@ def common_flags() -> List[CommonFlag]:
             type=regex_type,
             help="Regular expression that package attributes have to match (can be passed multiple times)",
         ),
+        CommonFlag(
+            "-P",
+            "--block-package",
```
Comment (Mic92) [inline nixpkgs_review/cli/__init__.py:161]:
```suggestion
            "--skip-package",
```

File: nixpkgs_review/cli/__init__.py [hunk 2]
```diff
@@ -156,6 +156,20 @@ def common_flags() -> List[CommonFlag]:
...
+            action="append",
+            default=[],
+            help="Packages to not build (can be passed multiple times)",
+        ),
+        CommonFlag(
+            "--block-package-regex",
```
Comment (Mic92) [inline nixpkgs_review/cli/__init__.py:167]:
```suggestion
            "--skip-package-regex",
```

**Comment by SuperSandro2000** [comment]
I am not sure why ``--skip-package-regex python`` does not match ``vscode-extensions.ms-python."vscode-pylance"`` but python packages.

File: nixpkgs_review/cli/__init__.py [hunk 3]
```diff
@@ -156,6 +156,20 @@ def common_flags() -> List[CommonFlag]:
             type=regex_type,
             help="Regular expression that package attributes have to match (can be passed multiple times)",
         ),
+        CommonFlag(
+            "-P",
+            "--skip-package",
```
Comment (Mic92) [inline nixpkgs_review/cli/__init__.py:161]: Can you also document these flags in the readme?

Comment (SuperSandro2000) [reply nixpkgs_review/cli/__init__.py:161]: done

File: nixpkgs_review/review.py [hunk 4]
```diff
@@ -367,6 +384,19 @@ def filter_packages(
         for regex in package_regexes:
             if regex.match(attr):
                 packages.add(attr)
+
+    if not packages:
+        packages = changed_packages
```
Comment (Mic92) [inline nixpkgs_review/review.py:389]: I don't understand this line. Packages is initialized in line 370.

Comment (SuperSandro2000) [reply nixpkgs_review/review.py]: Removed.

Comment (SuperSandro2000) [reply nixpkgs_review/review.py]: This is actually required and now I remember why. If we do not explicitly add a package we want to build packages is empty and we can't discard anything.

**Comment by Mic92** [comment]
Thanks!

**Comment by bobrik** [comment]
I'd love to see nixpkgs version support this 👍

**Comment by SuperSandro2000** [comment]
> I'd love to see nixpkgs version support this 👍

Master is currently unstable and tests need improvement so it could take a bit until it hits.

**Pull request merged**